hadoop-common-project/hadoop-auth/src/test/java/org/apache/hadoop/security/authentication/client/TestKerberosAuthenticator.java

```diff
@@ -89,7 +90,8 @@ private Properties getMultiAuthHandlerConfiguration() {
     return props;
   }
 
-  @Test(timeout=60000)
```

Comment:
It looks like this file (and a few additional ones) is using junit4 library for the most part.

Reply:
Thank you for your help with the review. You are right, many parts of this module are still using JUnit 4. We are actively upgrading from JUnit 4 to JUnit 5. The goal of this PR is to complete the upgrade of the hadoop-minikdc module and its unit tests, as well as to upgrade other modules that depend on hadoop-minikdc.